frontend/public/components/utils/file-input.tsx

```diff
@@ -1,16 +1,26 @@
 import * as React from 'react';
+import Dropzone from 'react-dropzone';
```

Comment:
Can you make sure usages of this module use the `AsyncComponent` pattern to ensure our bundle size isn't increased just for this one dependency?